ehr/resources/web/ehr/window/CreateTaskFromIdsWindow.js

```diff
@@ -234,7 +234,7 @@ Ext4.define('EHR.window.CreateTaskFromIdsWindow', {
         var type = this.down('#type').getValue().type;
         if (type == 'createNew'){
             return {
-                date: this.down('#date').getValue(),
```

Comment:
dont you want to respect 'date'? otherwise is it going to default to today's date?

Reply:
There is no 'date' column in ehr.tasks, there is a however a 'duedate' column, so this correction makes sense.

Reply:
OK, i buy that. In reality I would bet that 'date' is often left alone (so defaulting to today on the client-side), and the server-side is going to default duedate to today. I dont think duedate ends up being used for a lot in practice. That all said, this fix is probably the right thing.